Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -9,6 +9,10 @@
 /* Begin PBXBuildFile section */
 		00841DE524477805003CF74A /* AppTabBarDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = FFE891452445150B0058B642 /* AppTabBarDelegate.swift */; };
 		00841DE724477806003CF74A /* AppTabBarDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = FFE891452445150B0058B642 /* AppTabBarDelegate.swift */; };
+		00A7946B245CA4E60063BA18 /* ArticleSurveyTimerController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 00A7946A245CA4E60063BA18 /* ArticleSurveyTimerController.swift */; };
```

Comment:
this line is repeated 4 times. while I can't say for sure that's bad, that seems non-typical, right?

Reply:
These identifiers (`00A7946B245CA4E60063BA18` etc.) are actually not identical – easy to miss because they look so similar! I'm not an Xcode PBX file format expert, but if I'm not mistaken the similar looking lines are because of `ArticleSurveyTimerController`'s target membership in 4 app targets (e.g. Wikipedia, Staging, etc.), so this repetition is expected.

Reply:
It shows up because we have it selected for 4 different targets - Wikipedia, Staging, Experimental, and User Testing.

Reply:
Clearly I didn't look at those strings closely enough.